oval_graph/converter.py

```diff
@@ -29,6 +30,17 @@ def __init__(self, tree):
             "notappl": "glyphicon glyphicon-question-sign text-dark"
         }
 
+        self.HIDE_PASS_TESTS = {
+            True: {  # Key is value of hide_pass_tests option
+                "true": False,  # Key is result of node
+                "false": True
+            },
+            False: {
+                "true": True,
+                "false": True
+            }
+        }
```

Comment:
This is really interesting piece of code. I think that you can achieve your goal without this mind-boggling dictionary.